### PR TITLE
Upgrade to Bootstrap 5.1

### PR DESCRIPTION
**Pull request by simon04**
https://getbootstrap.com/docs/5.1/migration/

Screenshot 2022-05-20 at 23-16-48 Wikidata Image Positions

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -5,20 +5,22 @@
     <meta charset="utf-8">
     <title>{% block title %}Wikidata Image Positions{% endblock title %}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/5.1.3/css/bootstrap.min.css" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```
Comment (lucaswerkmeister) [inline templates/base.html]: Maybe we should change the name from `twitter-bootstrap` to just `bootstrap` while we’re at it? (Both seem to serve the exact same file, so I’m guessing `twitter-bootstrap` is the older name.)

**Review by lucaswerkmeister — COMMENTED** [review]
These buttons don’t look so great anymore:
image
Although to be fair, I only changed those to use `ml-2` (now `ms-2` apparently?) on Friday (eeb434ac0b) ^^

I’m not sure how I feel about the centered navbar-nav, but it’s probably okay.
image

File: templates/base.html (hunk 1 above)
Comment (simon04) [reply templates/base.html]: Good idea. Done.

**Comment by lucaswerkmeister** [comment]
> I’m not sure how I feel about the centered navbar-nav, but it’s probably okay. image

Ah, but your screenshot makes me realize that the documentation/toolforge/source part moves around depending on whether login is enabled or not (it’s either centered or on the end side, depending on whether there is an authentication area or not), that’s probably not ideal… I guess if OAuth isn’t configured, the authentication area should be something like an empty div instead of nothing at all? (I’m assuming that it looks the same as long as OAuth is configured, regardless of whether the user is logged in or not, but haven’t tried that out yet.)

**Comment by simon04** [comment]
Sorry, I missed the `mr-*` which are now called `me-*`. Should be fixed.

image

**Comment by lucaswerkmeister** [comment]
Deployed and looking good, thank you!

**Pull request merged**